### DISCUSSION TITLE
How to include font files in static directory?

### DISCUSSION THREAD

**Discussion by jamiescript** [jaredpalmer/razzle#1683] [Q&A] [answered]
## ❓Question

Under static directory I see 'js' and 'media' directories, but how do I push font files to static directory?

razzle.config.js
```
module.exports = {
  modifyWebpackConfig(opts) {
    const config = Object.assign({}, opts.webpackConfig);

    config.module.rules.push({
      test: /\.(otf|ttf).*$/,
      loader: 'url-loader?limit=10000',
    });
    return config;
  },
};
```

**Comment by fivethreeo** [comment, marked as answer]
Just put them in src/ and reference them in css url

**Comment by fivethreeo** [comment]
The static dir is copied directly to build/static and is meant for files that are not loaded by webpack i.e. robots.txt and favicon.ico
